casts/cards/src/UICollectionCard/useCurrentSourceItemIndex.ts

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { UIConstant } from '../constants';
+import type { MediaCardContent, MediaCardContentType } from '../types';
+
+function getShowTime(contentType: MediaCardContentType) {
+    switch (contentType) {
+        case 'Image':
+            return UIConstant.uiCollectionCard.timeToShowImage;
+        case 'Video':
+            return UIConstant.uiCollectionCard.timeToShowVideo;
+        case 'Unknown':
+        default:
+            return Infinity;
```

Comment:
I'm scared to return `Infinity` here as L31 would give an unpredicted result and will allocate the timeout forever (keeping it memory).

I'd better suggest to return `undefined` and do not create the timeout with an Infinity